lib/realtime/monitoring/erl_sys_mon.ex

```diff
@@ -56,4 +57,14 @@ defmodule Realtime.ErlSysMon do
     _ ->
       Logger.warning("#{__MODULE__} message: " <> inspect(msg))
   end
+
+  defp defensive_exit?(pid, pid_info, type) do
+    {{:"$initial_call", {mod, _, _}}, _} = pid_info
+
+    # credo:disable-for-next-line
+    cond do
+      mod == Realtime.Integration.WebsocketClient && type == :long_message_queue -> Process.exit(pid, :kill)
```

Comment:
This module is the one we use for testing right? 

Also for long_message_queue we might want to not kill on the current threshold (1000) but maybe if it goes beyond a certain number. Might be worth looking at some prod data?